website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+page_title: Upgrading to 1.10.0
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.10.0
+
+<EnterpriseAlert />
+
+## Introduction
+
+This guide explains how to best upgrade a single Consul Enterprise datacenter to v1.10.0
+from a version of Consul that is forwards compatible with v1.10. If you are on a major 
+version of Consul prior to 1.8, please follow the link for the version you are on from 
+[here](/docs/upgrading/instructions). If you are already on major versions 1.8 or 1.9
+this guide will go over the procedures required for upgrading to v1.10. If you have
+multiple Consul datacenters then they are upgraded in the normal order with each
+upgrade taking into account the instructions below.
+
+For the open source version of Consul please follow the 
+[General Upgrade Process](/docs/upgrading/instructions/general-process).
+
+~> You can only upgrade to Consul Enterprise 1.10 from a version of Consul Enterprise 
+1.8 >= 1.8.13 or 1.9 >= 1.9.7. Other versions of Consul Enterprise are not forwards
+compatible with v1.10 and will cause issues during the upgrade that could result in
+agents failing to start.
+
+## Requirements
+
+- All Consul servers, clients and snapshot agents should be on a version of Consul >= 1.8.0 and < 1.10.0.
+
+## Assumptions
+
+This guides makes the following assumptions:
```

Comment:
I would label this section header `#Prerequisites` and then list the prerequisites below.

Reply:
That would make more sense to me. However this format seemed standardized across all the upgrade instructions linked on this page https://www.consul.io/docs/upgrading/instructions.

Do you still think its worthwhile deviating in just this doc? Maybe its a change we want to make across all of them?

Reply:
ah I didn't realize this was some shared format, ok to leave this as is for now, it was a small nit.